zeroex/order_validator_test.go

```diff
@@ -22,88 +22,90 @@ var malformedAssetData = []byte("9HJhsAAAAAAAAAAAAAAAAInSSmtMyxtvqiYl")
 
-var testOrder = Order{
-	MakerAddress:          constants.GanacheAccount0,
-	TakerAddress:          constants.NullAddress,
-	SenderAddress:         constants.NullAddress,
-	FeeRecipientAddress:   common.HexToAddress("0xa258b39954cef5cb142fd567a46cddb31a670124"),
-	MakerAssetData:        common.Hex2Bytes("f47261b0000000000000000000000000871dd7c2b4b25e1aa18728e9d5f2af4c4e431f5c"),
-	TakerAssetData:        common.Hex2Bytes("f47261b00000000000000000000000000b1ba0af832d7c05fd64161e0db78e85978e8082"),
-	Salt:                  big.NewInt(1548619145450),
-	MakerFee:              big.NewInt(0),
-	TakerFee:              big.NewInt(0),
-	MakerAssetAmount:      big.NewInt(1000),
-	TakerAssetAmount:      big.NewInt(2000),
-	ExpirationTimeSeconds: big.NewInt(time.Now().Add(48 * time.Hour).Unix()),
-	ExchangeAddress:       constants.NetworkIDToContractAddresses[constants.TestNetworkID].Exchange,
+var testSignedOrder = SignedOrder{
+	Order: &Order {
```

Comment:
I'm sorry I'm just now noticing this. The embedded `Order` in `SignedOrder` should probably not be a pointer. We already pass around `SignedOrder` by reference pretty much everywhere so the additional indirection is not necessary and just introduces the possibility of bugs where `Order` could be nil.

Don't need to address now. I'll create a separate issue for it.